proposals/collection-expression-arguments.md

```diff
@@ -557,6 +563,6 @@ class MyCollection : IEnumerable
 }
 
 MyCollection c;
-c = [with(__arglist())];    // ok
-c = [with(__arglist(x, y)]; // ok
+c = [with(__arglist())];    // ok?
+c = [with(__arglist(x, y)]; // ok?
```

Comment:
Wg recommendation: no (unless it falls out as trivial). No one has asked for this, so no point putting in extra work with so much else that does matter. This is similar to conversations around dynamic support and the like.